resources/css/bem/chat-join-channel.less

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.chat-join-channel {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  background: hsl(var(--hsl-b4));
+
+  &__channel {
+    .reset-input();
+
+    display: grid;
+    grid-template-columns: subgrid;
+    grid-column: span 3;
+    color: hsl(var(--hsl-l3));
+    text-align: left;
+    padding: 5px 10px;
```

Comment:
looks like this causes overflow in firefox when the element type is button 🤨 seems fine with anchor or div at least